src/Aspire.Hosting.Python/PythonAppResourceBuilderExtensions.cs

```diff
@@ -357,20 +436,48 @@ private static IResourceBuilder<PythonAppResource> AddPythonAppCore(
             });
 
         // VS Code debug support - only applicable for Script and Module types
-        if (entrypointType is EntrypointType.Script or EntrypointType.Module)
+        if (supportDebugging)
```

Comment:
Executables can be run if they support being debugged as modules. AFAIK, this is most executables but I don't think we should make the assumption otherwise that all executables are _by default_ debuggable.